lib/active_admin/base_controller.rb

```diff
@@ -59,6 +59,10 @@ def active_admin_namespace
     end
     helper_method :active_admin_namespace
 
+    def status_code
+      ActionDispatch::ExceptionWrapper.new(env, env["active_admin.original_error"]).status_code
```

Comment:
Another problem is that when I hit the error page, I am going to see a 500 instead of a 404. This is because the routes to `namespace/error` exists. I have to think of something to stop this.